addon/helpers/register-select-helper.js

```diff
@@ -0,0 +1,16 @@
+import Ember from 'ember';
+
+export default function() {
+  Ember.Test.registerAsyncHelper('select', function(app, selector, text) {
+    const $el = app.testHelpers.findWithAssert(`${selector} option:contains("${text}")`);
+
+    $el.each(function() {
+      Ember.run(() => {
+        this.selected = true;
+        Ember.$(this).trigger('change');
+      });
+    });
```

Comment:
I decided to checkout this branch and see why Travis is hanging. It seems you can't use the fat arrow OR the string interpolation here. 

Refactored (and not as nice ES6 :frowning: ) code:
```js
var $el = app.testHelpers.findWithAssert(selector + "option:contains('" + text + "')");

$el.each(function() {
  var _this = this;

  Ember.run(function() {
    _this.selected = true;
    Ember.$(_this).trigger('change');
  });
});
```